IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcAlignmentSegment/Documentation.md

```diff
@@ -0,0 +1,7 @@
+An IfcAlignmentSegment is a segment of an IfcAlignment where either the vertical direction or horizontal direction or cant (in the case of trackdesign) obey a unique mathematical description as a function of chainage where chainage is the longitudinal distance along the horizontal projection of the whole alignment.
+
+Note 1 to entry: Unless otherwise stated, for railway, the appertaining track alignment design parameters are defined for the track centreline.
```

Comment:
```suggestion
>NOTE 1 to entry: Unless otherwise stated, for railway, the appertaining track alignment design parameters are defined for the track center line.
```